### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by atrol**

### CODE REVIEW

File: core/string_api.php [hunk 1]
```diff
@@ -721,75 +718,53 @@ function string_get_bug_view_url_with_fqdn( $p_bug_id ) {
...
 /**
- * return the name and GET parameters of a bug REPORT page for the given bug
- * account for the user preference and site override
- * @param integer $p_user_id A valid user identifier.
+ * return the name of a bug REPORT page
  * @return string
  */
-function string_get_bug_report_url( $p_user_id = null ) {
-	return string_get_bug_report_page( $p_user_id );
+function string_get_bug_report_url() {
```
Comment (vboctor) [inline core/string_api.php]: Why do we still need string_get_bug_report_url() and string_get_bug_report_page() rather than just one of them?

**Comment by vboctor** [comment]
:+1:

**Comment by dregad** [comment]
+1

File: core/string_api.php (hunk 1 above)
Comment (atrol) [inline core/string_api.php]: Removed string_get_bug_report_page

**Pull request merged**